singularity-eos/closure/mixed_cell_models.hpp

```diff
@@ -556,9 +557,18 @@ class PTESolverBase {
       temp[m] = 1.0;
       sie[m] = eos[m].InternalEnergyFromDensityTemperature(rho[m], Tguess, lambda[m]);
       // note the scaling of pressure
-      press[m] = robust::ratio(this->GetPressureFromPreferred(eos[m], rho[m], Tguess,
-                                                              sie[m], lambda[m], false),
-                               uscale);
+      if (eos.PreferredInput() & thermalqs::pressure) {
```

Comment:
Tests failing because of missing `[m]`
```suggestion
      if (eos[m].PreferredInput() & thermalqs::pressure) {
```